process/block/baseProcess.go

```diff
@@ -997,6 +996,7 @@ func (bp *baseProcessor) updateStateStorage(
 		return
 	}
 
+	accounts.CancelPrune(prevRootHash, data.NewRoot)
```

Comment:
prevRootHash or rootHash?

Reply:
Yes. There is an edge case if I cancelPrune on rootHash, and not on prevRootHash. If block n is finalized and I cancelPrune on rootHash, but the block that finalized the block n is roll backed, the cancelPrune will not be reverted.

Reply:
ok